### PR TITLE
#2839 Fix PIAntiWindUpTable

### PR DESCRIPTION
**Pull request by cossartque**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo [hunk 1]
```diff
@@ -44,13 +44,11 @@ model PIAntiWindUpTable "Proportional Integrator with anti-windup and table-base
...
+  if time >= 0 then
     y.value = combiTable1D.y[1];
-  elsewhen time >= 0 and u.value == pre(u.value) then
+  else
     y.value = pre(y.value);
-  elsewhen time < 0 then
-    y.value = 0;
-  end when;
+  end if;
   connect(add1.y, integrator.u) annotation(
```
Comment (ErwanGuichard49) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo:52]: According to the user's guide : One blank line between equations and a when equation or an if equation

Please add a blank line after end if;

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo [hunk 2]
```diff
@@ -44,13 +44,11 @@ model PIAntiWindUpTable "Proportional Integrator with anti-windup and table-base
   parameter Real Y0 "Start value of output";
 
 equation
-  when time >= 0 and u.value <> pre(u.value) then
+  if time >= 0 then
```
Comment (ErwanGuichard49) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo:47]: Have you checked that y.value is actualized each time combiTable1D.y[1] changes ?

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo [hunk 3]
```diff
@@ -44,13 +44,11 @@ model PIAntiWindUpTable "Proportional Integrator with anti-windup and table-base
...
     y.value = combiTable1D.y[1];
-  elsewhen time >= 0 and u.value == pre(u.value) then
+  else
     y.value = pre(y.value);
```
Comment (ErwanGuichard49) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo:50]: Does pre() work in an if structure ?

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo (hunk 1 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/PIAntiWindUpTable.mo:52]: done

**Pull request merged**